src/HOI4World/HoI4World.cpp

```diff
@@ -438,6 +450,14 @@ void HoI4::World::convertCountryNames(const Vic2::Localisations& vic2Localisatio
 				 *articleRules);
 		}
 	}
+	for (const auto& country: union_countries_)
+	{
+		hoi4Localisations->createCountryLocalisations(std::make_pair(country.GetOldTag(), country.GetTag()),
```

Comment:
What happens if the union country already has localizations in vanilla hoi4 or vic2? Do you pull those or craft own?

Reply:
The unions all have '_union' appended to their tags so they don't conflict.